source/troubleshooting.txt

```diff
@@ -16,95 +29,100 @@ When one attempts to connect to a <=3.4 version server, PyMongo will throw the f
   ...
   pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6).
 
-This is caused by the driver being too new for the server it is being run against.
-To resolve this issue either upgrade your database to version >= 3.6 or downgrade to PyMongo 3.x which supports MongoDB >= 2.6.
+This occurs when the driver is too new for the server it is connecting to.
+To resolve this issue, either upgrade your database to version >= 3.6 or downgrade to
+PyMongo 3.x which, supports MongoDB >= 2.6.
 
-
-'Cursor' object has no attribute '_Cursor__killed'
+'Cursor' Object Has No Attribute '_Cursor__killed'
 --------------------------------------------------
 
-On versions of PyMongo <3.9, when supplying invalid arguments the constructor of Cursor,
-there will be a TypeError raised, and an AttributeError printed to ``stderr``. The AttributeError is not relevant,
-instead look at the TypeError for debugging information:
+On versions of PyMongo <3.9, if you supply invalid arguments to the ``Cursor`` constructor,
```

Comment:
Suggesting the space here since there's a space in the paragraph above where it says "supports MongoDB >= 2.6" but I'm not sure which one is necessarily better (space or no space)


```suggestion
On versions of PyMongo < 3.9, if you supply invalid arguments to the ``Cursor`` constructor,
```